packages/zambdas/src/shared/pdf/external-labs-results-form-pdf.ts

```diff
@@ -206,23 +223,15 @@ export async function createLabResultPDF(
       // reportDate: '10/10/2024',
       // specimenSource: 'Throat',
       // specimenDescription: 'Throat culture',
-      specimenValue: undefined,
       specimenReferenceRange: undefined,
       resultPhase: diagnosticReport.status.charAt(0).toUpperCase() || ORDER_RESULT_ITEM_UNKNOWN,
       reviewed,
       reviewingProviderFirst: taskPractitioner.name?.[0].given?.join(',') || ORDER_RESULT_ITEM_UNKNOWN,
       reviewingProviderLast: taskPractitioner.name?.[0].family || ORDER_RESULT_ITEM_UNKNOWN,
       reviewingProviderTitle: ORDER_RESULT_ITEM_UNKNOWN,
       reviewDate: reviewDate,
-      results: observations.map((observation) => ({
-        resultCode: observation.code.coding?.[0].code || ORDER_RESULT_ITEM_UNKNOWN,
-        resultCodeDisplay: observation.code.coding?.[0].display || ORDER_RESULT_ITEM_UNKNOWN,
-        resultInterpretation: observation.interpretation?.[0].coding?.[0].code || ORDER_RESULT_ITEM_UNKNOWN,
-        resultInterpretationDisplay: observation.interpretation?.[0].coding?.[0].display || ORDER_RESULT_ITEM_UNKNOWN,
-        resultValue: `${observation.valueQuantity?.value || ORDER_RESULT_ITEM_UNKNOWN} ${
-          observation.valueQuantity?.code || ORDER_RESULT_ITEM_UNKNOWN
-        }`,
-      })),
+      resultInterpretations: resultInterpretationDisplays.length > 0 ? resultInterpretationDisplays : undefined,
```

Comment:
nit: having an empty array might be more robust solution. In this case, we can safely perform maps and other array methods on it and keep types simpler.

```
resultInterpretations: resultInterpretationDisplays,
```